src/App.jsx

```diff
@@ -7,7 +7,7 @@ function App() {
         <Router>
             <Routes>
                 <Route path="/" element={<Login/>}/>
-                <Route path="/login" element={<TodoList/>}/> {/* Página de destino */}
+                <Route path="/todos" element={<TodoList/>}/> {/* Página de destino */}
```

Comment:
proposed renaming. 

you are in the root, you get the login form. You submit the login credentials -> you access `/todos` the TODOs page.